src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10758,12 +10809,68 @@ definitions:
           The conditions that determine whether the actions will be executed for a rule.
 
           A condition should contain at least one entry in compositeConditions or simpleConditions.
+
+          If the field is omitted then the rule will apply to all nodes.
         type: '#/definitions/CompositeConditionDefinition'
       actions:
         description: The actions for the rule
         type: array
         items:
-          $ref: '#/definitions/ActionDefinition'
+          $ref: '#/definitions/ActionBodyExecTemplate'
+  RuleBody:
```

Comment:
The `RuleBody` is almost the same as `Rule`, isn't it possible to use one object in both cases? Or introduce inheritance?

Reply:
This wasn't possible last time I checked.  There's syntax for `allOf`, `anyOf` and `oneOf` which allow this sort of reuse, but IIRC there was some reason we couldn't use it.  @montgolfiere Do you remember any more about this issue?